src/pages/home/sidebar/ChatLinkRow.js

```diff
@@ -51,6 +51,12 @@ const ChatLinkRow = ({
         : styles.sidebarLinkText;
     const textUnreadStyle = option.isUnread
         ? [textStyle, styles.sidebarLinkTextUnread] : [textStyle];
+    const groupAvatar2DynamicStyles = !optionIsFocused
+        ? styles.chatSwitcherAvatar2ForGroupInactive
+        : styles.chatSwitcherAvatar2ForGroupActive;
+    const circularCount = (option.participants && option.participants.length > 2)
```

Comment:
It's not clear what this variable refers to. I'm thinking this is showing the "number of additional users count" to display? Maybe we can rename it to `additionalUsersCount`.

Reply:
Done.